src/scripts/main.js

```diff
@@ -1 +1,52 @@
 'use strict';
+
+Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject('First promise was rejected');
+  }, 3000);
+
+  document.body.addEventListener(
+    'click',
+    () => resolve('First promise was resolved'),
+    clearTimeout(timeoutId),
+    { once: true },
+  );
+})
+  .then((successResults) => createNotification(successResults, true))
+  .catch((errorResult) => createNotification(errorResult, false));
+
+Promise((resolve) => {
+  document.addEventListener('click', resolve('Second promise was resolved'));
+
+  document.addEventListener('contextmenu', () => {
+    resolve('Second promise was resolved');
+  });
+}).then((successResults) => createNotification(successResults, true));
```

Comment:
The 'resolve' function is being invoked immediately at the time of event listener registration. It should be wrapped in a function to be called when the event occurs, like '() => resolve(...)'. Also, you are not handling rejection for this promise.